Tests/WPLMNETClient.NetFramework.Tests/Tests.cs

```diff
@@ -18,19 +18,19 @@ public void OnStartUp()
         {
 
```

Comment:
We should prolly add a little  comment to the user the configuration and license object are specific to their website